docs/releases/PostAlpha2.1BranchMethodology.md

```diff
@@ -10,36 +10,38 @@ openroad app and also the submodules that it depends on. This
 methodology is efficient for developers and still allows us to maintain
 a stable externally facing branch.
 
-**Procedure :**
+## Procedure:
 
 The top level openroad app and all submodules which it depends on will
-develop in a branch called “openroad”. All repo’s included in the
-openroad app will use this branch name for consistency and so it’s easy
+develop in a branch called "openroad". All repo's included in the
```

Comment:
should be repositories or repos

Reply:
Will remove since this was kept only for history.